addon/styles/tailwind.config.js

```diff
@@ -137,7 +137,16 @@ module.exports = {
     */
 
     fontFamily: {
-      title: ['"Crimson Text"', 'serif'],
+      /*
+       * https://github.com/system-fonts/modern-font-stacks#old-style
+       */
+      title: [
+        '"Iowan Old Style"',
+        '"Palentino Linotype"',
+        '"URW Palladio L"',
+        '"P052"',
+        'serif',
```

Comment:
With `serif` fallback, it should work pretty much anywhere, but still - maybe its worth adding a windows font to the fallback list ?

Reply:
Per modern font stacks, `Palatino Linotype` is available on Windows XP and up.